shared/unattended/Fedora-25.ks

```diff
@@ -0,0 +1,45 @@
+install
+KVM_TEST_MEDIUM
+GRAPHICAL_OR_TEXT
+lang en_US
+keyboard us
+network --bootproto dhcp --hostname atest-guest
+rootpw 123456
+firewall --enabled --ssh
+selinux --enforcing
+timezone --utc America/New_York
+firstboot --disable
+bootloader --location=mbr --append="console=tty0 console=ttyS0,115200 serial"
+zerombr
+poweroff
+KVM_TEST_LOGGING
+
+clearpart --all --initlabel
+autopart
+
+%packages --ignoremissing
+@standard
+@c-development
+@development-tools
```

Comment:
No, not really.  This is the Fedora "full blown" image definition, which is intended to run many types of tests.  So, we've been adding development tools (compilers mainly) for a while (Fedora 21 has them, for example).